### DISCUSSION TITLE
「登录日志」的记录全都是服务器本机的 ip？

### DISCUSSION THREAD

**Discussion by nibazshab** [Mereithhh/vanblog#266] [General]
一般来说，不是应该记录登录时所使用的网络的IP地址吗？

网站是 docker 直接部署的，没有套 cdn，也没有反代

![{3EF9E5C6-A106-4366-BBE2-B38AC7A258A9}](https://user-images.githubusercontent.com/44338441/228259456-a280c2f8-1079-40f8-8b87-bddd2020729e.png)

**Comment by Mereithhh** [comment]
这个 ip 记录有点问题，我今天修一下。。

**Comment by owenzjg** [comment]
@Mereithhh 似乎这个问题还是没有修好

**Comment by CornWorld** [comment]
@nibazshab @owenzjg 目前还存在这样的问题吗

**Reply by nibazshab** [reply]
没有这个问题了
